### ISSUE TITLE
Multiple Select in subform

### ISSUE THREAD

**Issue by zalazdi** [kristijanhusak/laravel-form-builder#211]
If i add multiple select ```expanded=false``` it's generate code without ending brackes []. If i set ```expanded=true``` it's works.

```html
<select class="select2_multiple form-control" multiple="1" id="settings[sources]" name="settings[sources]">
[...]
</select>
```

```php
<?php

namespace App\Forms;

use App\Entities\Source;
use Kris\LaravelFormBuilder\Form;

class CampaignForm extends Form
{
    public function buildForm()
    {
        $this->add('name', 'text', [
            'label' => 'Nazwa',
            'help_block' => [
                'text' => 'Nazwa kampanii - na użytek wewnętrzny.',
            ],
        ]);
        $this->add('title', 'text', [
            'label' => 'Tytuł',
            'help_block' => [
                'text' => 'Tytuł wiadomości e-mail.',
            ],
        ]);
        $this->add('sender', 'text', [
            'label' => 'Nadawca',
            'help_block' => [
                'text' => 'Nadawca wiadomości. Bez adresu e-mail.',
            ],
        ]);
        $this->add('message', 'textarea', [
            'label' => 'Wiadomość',
            'help_block' => [
                'text' => 'Wiadomość w formacie HTML. Po dodaniu kampanii jest możliwość podglądu wiadomości.',
            ],
        ]);
        $this->add('amount', 'number', [
            'label' => 'Ilość wysyłek',
            'help_block' => [
                'text' => 'Maksymalna liczba wiadomości, jaką może dostać odbiorca z tej kampanii.',
            ],
        ]);
        $this->add('mah', 'text', [
            'label' => 'Podmiot odpowiedzialny',
        ]);

        $this->add('settings', 'form', [
            'class' => $this->settingsForm(),
            'name' => false,
            'formOptions' => [],

            'label' => false,
            'wrapper' => false,
        ]);

        $this->add('submit', 'submit', [
            'label' => ($this->model) ? 'Zapisz' : 'Dodaj',
            'attr' => [
                'class' => 'btn btn-default'
            ],
        ]);
    }

    public function settingsForm()
    {
        $form = $this->formBuilder->plain();

        $form->add('sources', 'choice', [
            'label' => 'Źródła',

            'choices' => Source::all()->pluck('name', 'id')->toArray(),
            'expanded' => false,
            'multiple' => true,
            'attr' => [
                'class' => 'select2_multiple form-control',
            ],
        ]);

        return $form;
    }
}
```

**Comment by kristijanhusak** [comment]
Thanks for reporting, i'll check it out.

**Comment by kristijanhusak** [comment]
@zalazdi I fixed this on `dev-master`, but i didn't published stable version yet. Hope it will be in few days.

**Comment by kristijanhusak** [comment]
Fixed in latest version (1.6.50). Thanks for reporting.

**Comment by yogesh7076** [comment]
how to use the Select2 Tagging with multi-value select boxes methode
